next.config.js

```diff
@@ -18,9 +18,8 @@ module.exports = withSourceMaps({
 	serverRuntimeConfig: {
 		rootDir: __dirname,
 	},
-	future: {
-		webpack5: true,
-	},
+	webpack5: true,
```

Comment:
@Lord-of-Codes what was the reason for this change?

Reply:
@sahilnanda1995 updated nextjs version to 11.0.1. In that webpack has come out of future. https://nextjs.org/docs/messages/future-webpack5-moved-to-webpack5